src/lib.rs

```diff
@@ -1680,7 +1681,7 @@ pub fn write_tx_tree(fi: &FrameInvariants, fs: &mut FrameState, cw: &mut Context
     let plane_bsize = get_plane_block_size(bsize, xdec, ydec);
 
     if bw_uv > 0 && bh_uv > 0 {
-        let uv_tx_type = tx_type;   // if inter mode, uv_tx_type == tx_type
+        let uv_tx_type = if has_coeff {tx_type} else {TxType::DCT_DCT}; // if inter mode, uv_tx_type == tx_type
```

Comment:
Oh, right- when luma tx-block has no coeffs, then tx_type for chroma should ne DCT_DCT.
I think I completely forgot this.